tutorials/Consistent_Scores.ipynb

```diff
@@ -0,0 +1,452 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Generate a list of thresholds of interest\n",
+    "thetas = murphy_thetas([fcst1, fcst2], obs, \"expectile\")\n",
+    "\n",
+    "# Calculate the average elementary score for the mean (0.5 expectile) for each threshold theta\n",
+    "ms1 = murphy_score(fcst1, obs, thetas, functional=\"expectile\", alpha=0.5)\n",
+    "ms2 = murphy_score(fcst2, obs, thetas, functional=\"expectile\", alpha=0.5)\n",
+    "\n",
+    "# Rename date variable for plotting\n",
+    "ms1 = ms1.rename({\"total\": \"Mean elementary score\", \"theta\": \"rainfall decision threshold (mm)\"})\n",
+    "ms2 = ms2.rename({\"total\": \"Mean elementary score\", \"theta\": \"rainfall decision threshold (mm)\"})\n",
+    "\n",
+    "# Plot the results\n",
+    "ms1[\"Mean elementary score\"].plot()\n",
+    "ms2[\"Mean elementary score\"].plot()\n",
+    "plt.title(\"fcst1 (blue), fcst2 (orange)\")\n",
+    "plt.suptitle(\"Murphy Score (for mean)\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Lower values are better on the Murphy Diagram. Fcst2 (orange) performs far worse at higher rainfall thresholds, so is penalised more heavily due to out weighting function $\\phi''(z) = \\exp ^\\frac{z}{10}$."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Example 2. A score consistent with a quantile\n",
+    "\n",
+    "Suppose we have a 90th percentile wind speed forecast that we want to evaluate, emphasising performance for higher thresholds. We can use the `consistent_quantile_score` in `scores`.\n",
```

Comment:
Suppose we have a 90th percentile wind speed forecast that we want to evaluate. Additionally, we wish to emphasise predictive performance for higher decision thresholds.